AncientReligions/events/ACR_other_event.txt

```diff
@@ -625,7 +625,7 @@ character_event = {
 
 			any_mercenary_band = {
 				limit = {
-					is_holy_order = yes
+					holy_order = yes
```

Comment:
reminder to myself, need to use a new version flag (instead of ancrel_carnutes) so this can fire again and work properly.